README.md

```diff
@@ -7,7 +7,7 @@ The scripts should work across a number of repos that include a `Dockerfile`. Th
 Including this repo within your own (as detailed below) will:
 * Build and tag the Docker image as detailed in the repo's `Dockerfile`. This happens on PR creation/update, tagging (including releases), and commits to the `master` branch
 * Push the image to the appropriate docker repo
-* If the Travis build has been triggered by a PR a new stack will be created within the `RANCHER_ENVIRONMENT` in Rancher. The rancher and docker templates used to build the stack are pulled from the [nhsuk/nhsuk-rancher-templates](https://github.com/nhsuk/nhsuk-rancher-templates) repo on GitHub. The specific compose files are determined by `RANCHER_TEMPLATE_NAME` and the latest subdirectory (i.e. the highest numbered). This is the convention used by rancher catalogs to implement template versioning.
+* If the Travis build has been triggered by a PR a new stack will be created within the `RANCHER_ENVIRONMENT` on the Rancher server based on the rancher template defined by RANCHER_TEMPLATE_NAME. The templates are defined in the Github repo [nhsuk/nhsuk-rancher-templates](https://github.com/nhsuk/nhsuk-rancher-templates).
```

Comment:
The version is now determined by an API call. I believe the value returned is the version specified in the config.yml (e.g. https://github.com/nhsuk/nhsuk-rancher-templates/blob/master/templates/c2s-gp-finder/config.yml).

@adamgraves-choices - can you confirm this?